cliquet/resource.py

```diff
@@ -282,7 +290,8 @@ def collection_delete(self):
         filters = self._extract_filters()
         deleted = self.collection.delete_records(filters=filters)
 
-        return self.postprocess(deleted)
+        action = len(deleted) > 0 and ACTION_DELETE or ACTION_READ
```

Comment:
this use case is very cool but I think we should narrow the scope of this PR to the use case, and keep in mind your proposal for the next iterations